kannon/master.py

```diff
@@ -69,9 +74,13 @@ def build(self, root_task: gokart.TaskOnKart) -> None:
                 continue
             # execute task
             if isinstance(task, TaskOnBullet):
+                if self.max_child_jobs is not None and len(running_task_ids) >= self.max_child_jobs:
```

Comment:
If `max_child_jobs` is specified as `0` by the user and `task` is an instance of `TaskOnBullet`, the `task` instance will never be processed, which makes the program never stops.
My proposal:
- Validate `max_child_jobs` is `None` or `>=1`

Reply:
I added https://github.com/m3dev/kannon/pull/24/commits/f2bf4adb9fcb18a71a1a6ef715bf77591e292e06 :+1: